docs/ehrql-new-tutorial-intro.md

```diff
@@ -0,0 +1,297 @@
+# ehrQL tutorial
+
+---8<-- 'includes/data-builder-danger-header.md'
+
+## Table of contents
+
+!!! todo
+
+    Remove this.
+
+    We should incorporate this as a full part of the documentation,
+    so that it is easy to discover and follow on to the next page.
+
+    This will be when this documentation is ready
+    (or close to ready)
+    for public use.
+
+* [Introduction (this page)](ehrql-new-tutorial-intro.md)
+* [Dataset definition 1a](ehrql-new-tutorial-1a.md)
+* [Dataset definition 1b](ehrql-new-tutorial-1b.md)
+* [Dataset definition 2a](ehrql-new-tutorial-2a.md)
+* [Unwritten remaining section placeholders](ehrql-new-tutorial-remaining-sections.md)
+
+## Audience
+
+This tutorial is aimed at:
+
+* **new users of Data Builder**
+  who wish to write dataset definitions
+  to extract data from OpenSAFELY backends
+* **developers involved in operating data backends**
+  who wish to understand more about how OpenSAFELY backends
+  are accessed by users
+
+!!! tip
+    There is no requirement for approval or access to OpenSAFELY to follow this tutorial.
+    This tutorial uses artificial data throughout.
+
+## What this tutorial covers: writing and running ehrQL queries with Data Builder
+
+ehrQL is a query language for electronic health records.
+The [introduction to ehrQL](ehrql-intro.md) has more information.
+
+Data Builder is a program that takes ehrQL queries,
```

Comment:
I understood this as "a programme of work" which created a pause in my parsing.

Perhaps "is a software component" or just "is software"?

Reply:
Separately - I find "Data Builder" hard to parse. This is just a note that perhaps we should revisit naming of components one more time when user testing things like names and syntax with real people.